packages/devextreme/js/__internal/ui/chat/messagebox.ts

```diff
@@ -154,6 +150,18 @@ class MessageBox extends DOMComponent<MessageBox, Properties> {
     this._button.option('disabled', state);
   }
 
+  _updateAria(emptyViewId: string | null): void {
```

Comment:
Because we use this method outside of messagebox, its name shouldn't include underscore prefics

Reply:
Maybe we can specify a concrete method name since we are only updating aria-labelledby here?

Reply:
[ddd98c2f8675ba0e9ea72a8e3cde9824787a94a4]